usr/share/rear/backup/DUPLICITY/default/500_make_duplicity_backup.sh

```diff
@@ -21,41 +21,37 @@ if [ "$BACKUP_PROG" = "duplicity" ] ; then
     # by falk hoeppner
 
     if [ -n "$BACKUP_DUPLICITY_ASK_PASSPHRASE" ]; then
```

Comment:
I think BACKUP_DUPLICITY_ASK_PASSPHRASE is a boolean variable.
For boolean variables better use the functions `is_true` or `is_false`,
see the comments at the function definitions in lib/global-functions.sh
how to use them - in particular `! is_true` is not the same as `is_false`
and `! is_false` is not the same as `is_true` because 'tertium datur' here.